### PR TITLE
Please add my lesson 9 homework

### PR DESCRIPTION
**Pull request by zoldirazvan**

### CODE REVIEW

**Comment by Liitle** [comment]
Hi, you committed the changes on branch lesson_9 and made the pull request from lesson_9 to master.

Please switch to a new branch `lesson_9_razvan_zoldi` and open a new pull request from the new branch to `lesson_9`. After you move the code you can close this pullrequest

Let me know when you moved the code and I will clean-up the `lesson_9` branch

File: lesson_9/lesson_9_zoldi_razvan/lesson_9_zoldi_razvan.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                   f'and same area: {self.position}'
+        elif self.continent is ref_country[1] and self.position is not ref_country[2]:
+            return f'Both, {ref_country[0]} and {self.name}, are on same continent ({self.continent}) ' \
+                   f'but on different areas: \n{ref_country[0]} is on {ref_country[2]} area ' \
+                   f'and {self.name} is on {self.position} area'
+        elif self.continent is not ref_country[1]:
+            return f' {ref_country[0]}  and {self.name} are on different continents'
+
+    def area_comparison(self):
+        if int(self.area) == int(ref_country[3]):
```
Comment (Liitle) [inline lesson_9/lesson_9_zoldi_razvan/lesson_9_zoldi_razvan.py]: You can avoid the int conversion if you initialize the instances directly with int values

**Review by Liitle — COMMENTED** [review]
Nice work.

**Comment by zoldirazvan** [comment]
> Hi, you committed the changes on branch lesson_9 and made the pull request from lesson_9 to master.
> 
> Please switch to a new branch `lesson_9_razvan_zoldi` and open a new pull request from the new branch to `lesson_9`. After you move the code you can close this pullrequest
> 
> Let me know when you moved the code and I will clean-up the `lesson_9` branch

Hi, 
In my branch appear two other folders as you can see on attachment. So is possible to commit 2 other's code. I don't understand. 
image

**Review by mariuspodean — CHANGES_REQUESTED** [review]
Please clean the commit so that it includes only the added files (you are also deleting some other files)

**Comment by zoldirazvan** [comment]
> Please clean the commit so that it includes only the added files (you are also deleting some other files)

I'm not sure how to do that, that is why I'm hesitating. Maybe is wise to delete the commit and make another one.

**Comment by mariuspodean** [comment]
> > Please clean the commit so that it includes only the added files (you are also deleting some other files)
> 
> I'm not sure how to do that, that is why I'm hesitating. Maybe is wise to delete the commit and make another one.

True, here probably it would be easier to make a new branch and add only the new file

**Pull request closed without merging**